Cargo.toml

```diff
@@ -34,57 +34,55 @@ debug = false
 incremental = false
 
 [workspace.dependencies]
-signet-blobber = { version = "0.10", path = "crates/blobber" }
-signet-block-processor = { version = "0.10", path = "crates/block-processor" }
-signet-db = { version = "0.10", path = "crates/db" }
-signet-node-types = { version = "0.10", path = "crates/node-types" }
-signet-rpc = { version = "0.10", path = "crates/rpc" }
-
-init4-bin-base = { version = "0.11.0", features = ["alloy"] }
-
-signet-bundle = "0.10.1"
-signet-constants = "0.10.1"
-signet-evm = "0.10.1"
-signet-extract = "0.10.1"
-signet-journal = "0.10.1"
-signet-tx-cache = "0.10.1"
-signet-types = "0.10.1"
-signet-zenith = "0.10.1"
+signet-blobber = { version = "0.9.0", path = "crates/blobber" }
+signet-db = { version = "0.9.0", path = "crates/db" }
+signet-node-types = { version = "0.9.0", path = "crates/node-types" }
+signet-rpc = { version = "0.9.0", path = "crates/rpc" }
+
+init4-bin-base = { version = "0.10.0", features = ["alloy"] }
+
+signet-bundle = "0.11.0"
+signet-constants = "0.11.0"
+signet-evm = "0.11.0"
+signet-extract = "0.11.0"
+signet-tx-cache = "0.11.0"
+signet-types = "0.11.0"
+signet-zenith = "0.11.0"
 
 # ajj
 ajj = { version = "0.3.4" }
 
 # trevm
-trevm = { version = "0.27.8", features = ["full_env_cfg"] }
+trevm = { version = "0.29.0", features = ["full_env_cfg"] }
 
 # Alloy periphery crates
-alloy = { version = "1.0.25", features = [
+alloy-core = { version = "1.4" }
```

Comment:
this should be unnecessary as it specified in sdk?

Reply:
yeah, leftover from patches